config/src/config/utils.rs

```diff
@@ -34,3 +35,33 @@ impl RootPath {
         }
     }
 }
+
+/// Returns true iff failpoints are enabled
+pub fn are_failpoints_enabled() -> bool {
+    cfg_if! {
```

Comment:
What's the difference between this and just calling `if cfg!(feature = "failpoints)`, and just calling it on the call site?  -- I feel it's unnecessarily verbose.

Reply:
It's identical 😄 I've just wrapped it in the function `are_failpoints_enabled` so that it can be used across the sanitizer and optimizer (to avoid duplication). Does that make sense?